src/main/java/io/anserini/reproduce/RunMsMarco.java

```diff
@@ -66,14 +66,14 @@ public static void main(String[] args) throws Exception {
     }
     switch (MsMarcoArgs.MsMarcoVersion) {
         case 2:
-            COLLECTION = "msmarco-v2.1-doc";
+            COLLECTION = "msmarco-v2.1";
```

Comment:
can we be consistent here and change the parameter to either "msmarco-v1-passage" or "msmarco-v2.1" as opposed to "1" and "2"? We'll add others later...